### ISSUE TITLE
No tray icon on Ubuntu 17.10

### ISSUE THREAD

**Issue by tmortagne** [transmission-remote-gui/transgui#1132]
I have both "Minimize to tray" and "Tray icon always visible" but they don't seems to have any effect.

It's working well for Steam and HPLB at least so I deffinitely have a tray icon area but the way transgui does it does not seems to work.

Just in case it related in any way: I have a bunch of the following error printed in the console when starting transgiu from a terminal: 

```
(transgui:15192): Gtk-CRITICAL **: IA__gtk_widget_realize: assertion 'GTK_WIDGET_ANCHORED (widget) || GTK_IS_INVISIBLE (widget)' failed
```

**Comment by x-dragos** [comment]
This also manifests in Ubuntu 18.4

**Comment by leonsoft-kras** [comment]
Try to contact the lazarus / ubuntu community. We can't support 100 versions of Linux. 
Perhaps there will be a solution.

**Comment by tmortagne** [comment]
> We can't support 100 versions of Linux.

I agree and you can definitely forget about 17.10 but Ubuntu 18.04 (the LTS version for the next two years) is not exactly an exotic distribution :)

**Comment by leonsoft-kras** [comment]
Changes in the code make 2 people maximum. The main operating system is windows. I have raspberry pi. It's all.

Linux lovers love free code, but do not want to modify it for free. And I'm 51% sure that it's about implementing the code in ubuntu.

**Comment by cociweb** [comment]
+1
anyway, as a workaround:
create a file eg. transgui.desktop in /usr/share/applications/ with
`[Desktop Entry]
Name=Transgui
Comment=Torrent and magnetlink downloader
Terminal=false
Icon=/etc/transgui/transgui.png
Exec=/etc/transgui/transgui
Type=Application
MimeType=application/x-bittorrent;x-scheme-handler/magnet;
Categories=Application;Utility;Network;`
where Icon and Exec points to your transgui path
also add into /usr/share/applications/mimeinfo.cache the following lines:
`application/x-bittorrent=transgui.desktop;
x-scheme-handler/magnet=transgui.desktop;`

The icon and file association works for me under 18.04.
